Integrations/integration-McAfee-ePO.yml

```diff
@@ -262,32 +375,20 @@ script:
             } else {
                 md += 'No systems found\n';
             }
-            return {Type: entryTypes.note, Contents: result, ContentsFormat: formats.json, HumanReadable: md};
+            return {
+                Type: entryTypes.note,
+                Contents: result,
+                ContentsFormat: formats.json,
+                HumanReadable: md,
+                EntryContext: {
+                    'Endpoint(val.IPAddress==obj.IPAddress)': prettifyFindSystem(result),
+                    'McAfee.ePO.Endpoint(val.IPAddress==obj.IPAddress)': prettifyFindSystem(result),
```

Comment:
There is no point to duplicate the outputs unless `McAfee.ePO.Endpoint` has more specific fields related to McAfee . So `Endpoint` need to have only the base fields according to our context structure conventions (with the correct names) and the `McAfee.ePO.Endpoint` should contain all the fields

Reply:
There are, i'll add more fields to McAfee.ePO